sdk/translation/ai-translation-document-rest/src/isUnexpected.ts

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+// Copyright (c) Microsoft Corporation.
```

Comment:
Why do we have two headers here?

Reply:
This will be emitter-related too. Might be related to the change in copyright header casing (`License` vs `license`) and the emitter isn't picking up the fact that the header is already present anymore?

I suspect this will go away once @jeremymeng's PR to the generator (https://github.com/Azure/autorest.typescript/pull/2775) is released.